src/SDKs/CognitiveServices/dataPlane/Search/BingCustomImageSearch/BingCustomImageSearch/Microsoft.Azure.CognitiveServices.Search.CustomImageSearch.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+  
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.Search.CustomImageSearch</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services Custom Image Search API.</Description>
+    <VersionPrefix>1.2.0</VersionPrefix>
```

Comment:
Is this a brand new package? 
If yes, please start versioning from `1.0.0`
Is this preview?
If yes, please start versioning from `0.9.0-preview`

Reply:
Updated